<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_DPSV_filter</class>
 <widget class="QDialog" name="dlg_DPSV_filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Neighboorhood Model</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>2D - neighboorhood</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>3D - neighboorhood</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QRadioButton" name="m_pNeighb4">
         <property name="text">
          <string>4</string>
         </property>
         <attribute name="buttonGroup">
          <string>m_pButtonGroup</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="m_pNeighb8">
         <property name="text">
          <string>8</string>
         </property>
         <attribute name="buttonGroup">
          <string>m_pButtonGroup</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="m_pNeighb6">
         <property name="text">
          <string>6</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>m_pButtonGroup</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="m_pNeighb26">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>26 </string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string>m_pButtonGroup</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Mask Size</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="m_pMaskSize">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Path Length            </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="m_pPathLength">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>11</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Beta</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="m_pBeta">
          <property name="text">
           <string>0.001</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <item>
      <layout class="QHBoxLayout" name="_2">
       <item>
        <widget class="QPushButton" name="m_pOKButton">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="m_pButtonGroup"/>
 </buttongroups>
</ui>
